tests/integration/sniffer/capturer.py

```
@@ -0,0 +1,858 @@
+#!/usr/bin/env python3
+
+import os
+import logging
+import time
+import shlex
+from typing import Dict, Set, Optional, Any
+from datetime import datetime, timezone
+from scapy.all import rdpcap, IP, TCP, UDP, SCTP
+from scapy.layers.http import HTTP, HTTPRequest, HTTPResponse
```

Ruff (0.12.2)

`scapy.layers.http.HTTP` imported but unused

Remove unused import: `scapy.layers.http.HTTP`

(F401)